saleor/graphql/core/utils/__init__.py

```diff
@@ -36,6 +36,10 @@ def str_to_enum(name):
 
 def validate_image_file(file, field_name):
     """Validate if the file is an image."""
+    if not file:
```

Comment:
Type hint?

Reply:
I tried `UploadedFile`, and `InMemoryUploadedFile`
but mypy rasied `Item "None" of "Optional[str]" has no attribute "startswith"`

Reply:
Probably because you need to put it as `Optional`?

Reply:
Or maybe because `content_type` is nullable (implicitly/explicitly)?

Reply:
`content_type` can be None